### PR TITLE
include error message when response validation catches a failed API call

### PR DESCRIPTION
**Pull request by abelcastilloavant**
When looker responds to a failed request, it gives you a message with some clue as to why the request failed. This PR extracts that message and drops it in the error message coming from `validate_response`.

### CODE REVIEW

File: R/response_handlers.R [hunk 1]
```diff
@@ -17,9 +17,11 @@ validate_response <- function(response) {
...
   stop(paste("The",
     gsub("_", " ", deparse(substitute(response))),
-    "of your Looker query was not a successful response.",
-    "It returned a status code of",
-    httr::status_code(response)
+    "step in looker failed.",
+    "The status code was",
+    httr::status_code(response),
+    "and the error message was",
+    httr::content(response)$message
```
Comment (peterhurford) [inline R/response_handlers.R]: Would there ever not be a `$message`, causing this to break?

**Comment by khalidharun** [comment]
## [Current coverage][cc-pull] is **75.80%**
> Merging [#29][cc-pull] into [master][cc-base-branch] will increase coverage by **0.80%**

```diff
@@             master        #29   diff @@
==========================================
  Files             5          5          
  Lines            60         62     +2   
  Methods           0          0          
  Messages          0          0          
  Branches          0          0          
==========================================
+ Hits             45         47     +2   
  Misses           15         15          
  Partials          0          0          
```

> Powered by [Codecov](https://codecov.io?src=pr). Last updated by [f0e8c11...ffc0013][cc-compare]
[cc-base-branch]: https://codecov.io/gh/avantcredit/looker3/branch/master?src=pr
[cc-compare]: https://codecov.io/gh/avantcredit/looker3/compare/f0e8c11ca1821d7122bbe5f0a68b617441ed8ef7...ffc0013dc666fa4ea64762dcf8fef1d06d0a44da
[cc-pull]: https://codecov.io/gh/avantcredit/looker3/pull/29?src=pr

File: tests/testthat/test-response_handlers.R [hunk 2]
```diff
@@ -11,12 +11,14 @@ fake_query_failure   <- list(status = "500")
...
+with_mock(
+  `httr::status_code` = function(x) x$status, 
+  `httr::content`     = function(...) list(message = "error message"), {
   test_that("validate_response errors with step name and status code", {
     expect_error(validate_response(fake_query_failure),
-      "The fake query failure of your Looker query")
+      "The fake query failure step in looker")
     expect_error(validate_response(fake_query_failure),
-      "returned a status code of 500")
+      "status code was 500")
```
Comment (peterhurford) [inline tests/testthat/test-response_handlers.R]: should we test the `message` portion as well?

**Comment by abelcastilloavant** [comment]
tagged and released

**Pull request merged**